lib/pbench/server/api/resources/query_apis/datasets_detail.py

```diff
@@ -76,7 +80,7 @@ def assemble(self, json_data: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
                         "bool": {
                             "filter": [
                                 {"match": {"run.name": name}},
-                                {"match": self._get_user_term(user)},
+                                {"match": self._get_user_term(json_data)},
                             ]
```

Comment:
Why does this filter use `match` (i.e., for the `"user"`) where the other two use `term`?

Reply:
That's actually a good question, and really it's just the way it ended up; I think the old V1 queries may have been "match", but that didn't have the strict separation between "keyword" and "text" (they were all "string"). These are all keyword fields, so I don't think it really makes a difference, although there's a possibility that "term" would be more efficient.